src/common/fmt_common.h

```diff
@@ -38,11 +38,19 @@ template<class T>
 concept has_alt_fmt_print = requires(T t) {
   { t.alt_fmt_print(bool{}) } -> std::same_as<std::string>;
 };
+#if FMT_VERSION >= 110000
 template<class T>
 concept has_fmt_print_ctx = requires(
-  T t, fmt::buffer_context<char> &ctx) {
+  T t, fmt::buffered_context<char> &ctx) {
   { t.fmt_print_ctx(ctx) } -> std::same_as<decltype(ctx.out())>;
 };
```

Comment:
That's probably a mistake on my side. One of these should have the old format.

Reply:
Ack